### PR TITLE
Updated Mother building artwork

### PR DESCRIPTION
**Pull request by Dzierzan**
mother

### CODE REVIEW

File: mods/hv/maps/shellmap/map.yaml [hunk 1]
```diff
@@ -686,11 +679,6 @@ Actors:
 		Location: 88,22
 		Faction: Random
 		Owner: Synapol Corporation
-	Actor246: storage
-		Location: 69,13
-		Faction: Random
-		Owner: Synapol Corporation
-		FreeActor: False
```
Comment (Mailaender) [inline mods/hv/maps/shellmap/map.yaml:693]: This looks like an accidental change.

Comment (Dzierzan) [reply mods/hv/maps/shellmap/map.yaml:693]: Ohh, not, not, I actually not, building was wrong. I mean YI storage was in a place of SC storage. But I again forgot to disable FreeActor for that. Let me fix it.

Comment (Mailaender) [reply mods/hv/maps/shellmap/map.yaml:693]: This is still `storage` though, you want `storage2`?

Comment (Dzierzan) [reply mods/hv/maps/shellmap/map.yaml:693]: No such actor as `storage2` ^^. It uses:

`	RenderSprites:
		PlayerPalette: green
		Image: storage2
		FactionImages:
			sc: storage`

Comment (Mailaender) [reply mods/hv/maps/shellmap/map.yaml:693]: Yes, but this is identical except for `Faction: Random` is that the actual wanted change here?

Comment (Dzierzan) [reply mods/hv/maps/shellmap/map.yaml:693]: And what's the issue? Considering nearly every actor has `Faction: Random` in Shellmap?

**Pull request merged**